src/main/flight/mixer.c

```diff
@@ -526,6 +548,13 @@ void writeMotors(void)
         }
 #endif
         for (int i = 0; i < motorCount; i++) {
+            if (revFlags[i]) {
+                if (motor[i] == disarmMotorOutput) {
+                    motor[i] = motorOutputHigh;
+                } else {
+                    motor[i] = (motorOutputHigh + motorOutputLow) - motor[i];
```

Comment:
This makes little sense for motors - in normal mode, the result will be catastrophic (full throttle), for 3d mode, it will work only is high/low is symmetric against neutral throttle (but it seems that BF code works this way)

Whole mixer code (and 3d handling) is already terribly complicated. Adding new special cases into it will eventually make it unmaintainable ..

Reply:
Agreed!